app/views/passenger_mailer/notify_pending.haml

```diff
@@ -0,0 +1,6 @@
+Dear #{@passenger.name},
+
+Your account with the Special Transportation Service has been created but is still pending. You can receive rides until (three business days from now, will insert actual date).
```

Comment:
You should replace `(three business days from now, will insert actual date)` with the actual date three business days from whenever "now" is.